src/Tracy/Bar.php

```diff
@@ -105,9 +129,49 @@ public function render()
 			}
 		}
 		$session = NULL;
-
+		ob_start();
 		$info = array_filter($this->info);
-		require __DIR__ . '/templates/bar.phtml';
+		require __DIR__ . '/templates/bar.content.phtml';
+		$content = ob_get_clean();
+		if ($this->handle) {
+			fwrite($this->handle, $content);
+			fclose($this->handle);
+		}
+
+		if (!connection_aborted() && Helpers::isHtmlMode()) {
+			if ($this->handle) {
+				$id = $this->id;
+			}
+			require __DIR__ . '/templates/bar.phtml';
+		}
+	}
+
+
+	private function dispatch($command)
+	{
+		if (preg_match('#^content.(\w+)$#', $command, $m)) {
+			$directory = $this->directory ?: ini_get('upload_tmp_dir');
+			$file = ($this->directory ?: ini_get('upload_tmp_dir')) . '/bar.' . $m[1];
+			header('Content-Type: text/javascript');
+			if ($content = @file_get_contents($file)) {
+				echo 'Tracy.Debug.newContent(' . json_encode(Helpers::fixEncoding($content)) . ');';
```

Comment:
Ideally. When I developed panel on current Tracy, sometimes I made mistake and it was hard to find where in such one-line code. I helped myself to dump it by var_dump() but this would be more developer friendly.